tests_notebooks/conftest.py

```diff
@@ -41,25 +41,39 @@ def docker_compose(docker_services):
 
 
 @pytest.fixture(scope="session")
-def aiidalab_exec(docker_compose):
-    def execute(command, user=None, **kwargs):
-        workdir = "/home/jovyan/apps/home"
-        if user:
-            command = f"exec --workdir {workdir} -T --user={user} aiidalab {command}"
-        else:
-            command = f"exec --workdir {workdir} -T aiidalab {command}"
+def aiidalab_exec(docker_compose, nb_user):
+    """Execute command inside the AiiDAlab test container"""
 
+    def execute(command, user=None, **kwargs):
+        workdir = f"/home/{nb_user}/apps/home"
+        if user is None:
+            user = nb_user
+        command = (
+            f"exec --workdir {workdir} -T --user={user} aiidalab bash -c '{command}'"
```

Comment:
This is the important fix, we need to escape the commands passed to docker compose, using `bash -c '<command>'`